nipoppy_cli/nipoppy/workflows/base.py

```diff
@@ -70,6 +70,13 @@ def __init__(
         self.logger = logger
         self.dry_run = dry_run
 
+        self.is_datalad_dataset = False
+        if is_datalad_installed:
+            logging.getLogger("datalad").setLevel(logging.WARNING)
+            logging.getLogger("datalad.gitrepo").setLevel(logging.ERROR)
+            if (self.dpath_root / ".datalad").exists():
+                self.is_datalad_dataset = True
```

Comment:
silence some of the datalad output log

should probably be adapted to the verbosity level

also there are still oodles of datalad log still printed out